code/datums/abilities/zoldorfabilities.dm

```diff
@@ -171,7 +171,7 @@
 					sentence += "[speechinput] "
 					speechinput = input("[sentence](noun) and (verb) yourself.", "Noun", null) as null|anything in fortune_nouns
 					if(!speechinput) break
-					sentence += "[speechinput] and "
+					sentence += "[speechinput] "
 					speechinput = input("[sentence]and (verb) yourself.", "Verb", null) as null|anything in fortune_verbs
```

Comment:
they're complaining about the display, not the actual resulting construction - the `and` needs to be shifted to the next addition instead

Reply:
I see.. the code here is a bit of a mindfuck

Reply:
Should be good now, but I can verify later